src/dialog/index.ts

```diff
@@ -172,8 +171,8 @@ export class DialogBase<P extends DialogProperties = DialogProperties> extends T
 			aria = {},
 			closeable = true,
 			closeText,
-			enterAnimation = animations.fadeIn,
-			exitAnimation = animations.fadeOut,
+			enterAnimation = this.theme(css.enter) || undefined,
```

Comment:
Another thought -- would it make sense to add `enterAnimationActive` and `exitAnimationActive` properties to support both transitions and animations for increased styling flexibility? Using keyframes for a simple opacity fade in/out seems overkill, although I suppose any given transition could be made into an animation. Your call.